charts/kargo/values.yaml

```diff
@@ -88,11 +88,11 @@ api:
   adminAccount:
     ## @param api.adminAccount.enabled Whether to enable the admin account.
     enabled: true
-    ## @param api.adminAccount.passwordHash Bcrypt password hash for the admin account. If specified, will ignore `api.adminAccount.password`.
+    ## @param api.adminAccount.passwordHash Bcrypt password hash for the admin account. If specified, will ignore `password`. A value MUST be provided for either this field or `password`.
```

Comment:
```suggestion
    ## @param api.adminAccount.passwordHash Bcrypt password hash for the admin account. If specified, will ignore `password`. A value **must** be provided for either this field or `password`.
```

Reply:
I don't have super strong feelings about this, but all caps MUST is a conventional way of calling out absolute requirements in technical docs -- most often seen if RFCs.

Reply:
I switched them to markdown bold to keep it consistent since I had done the same in the switch to the generated readme PR. So we need to choose our (the project's) preference

Reply:
I didn't realize it was already the other way in other places. I'm fine with this suggestion then.